CIME/case/case_post_run_io.py

```diff
@@ -0,0 +1,113 @@
+"""
```

Comment:
I think that this file can also be located in scorpio and it makes more sense to have it there.   I have a couple of examples of how to do this if you are interested.

Reply:
Sure, please let me know.

Reply:
@jedwards4b said it could go in scorpio. I'm not sure how to make that happen such that python knows to search that directory for imports. My thought would be to put it in `E3SM/cime_config/customize`

Reply:
I must admit that I would need to experiment a bit to come up with a functional solution.

Reply:
ok, I will try moving the file to $E3SM_SRC/cime_config/customize for now

Reply:
case_post_run_io.py has been moved out of CIME to $E3SM_SRC/cime_config/customize